biosimulators_utils/config.py

```diff
@@ -6,16 +6,25 @@
 :License: MIT
 """
 
-from .omex_meta.data_model import OmexMetadataInputFormat, OmexMetadataOutputFormat, OmexMetadataSchema
-from .report.data_model import ReportFormat  # noqa: F401
-from .viz.data_model import VizFormat  # noqa: F401
-from kisao import AlgorithmSubstitutionPolicy  # noqa: F401
-import appdirs
 import enum
 import os
+import platform
+from typing import Dict, List, Union
+from kisao import AlgorithmSubstitutionPolicy  # noqa: F401
+from biosimulators_utils.log.data_model import StandardOutputErrorCapturerLevel
+from biosimulators_utils.omex_meta.data_model import OmexMetadataInputFormat, OmexMetadataOutputFormat, OmexMetadataSchema
+from biosimulators_utils.report.data_model import ReportFormat  # noqa: F401
+from biosimulators_utils.viz.data_model import VizFormat  # noqa: F401
+import appdirs
 
 __all__ = ['Config', 'get_config', 'Colors', 'get_app_dirs']
 
+CURRENT_PLATFORM = platform.system()
+try:
+    assert CURRENT_PLATFORM == "Darwin"
```

Comment:
so this will catch all Mac OS? Have you also tested the bug in non-Mac OS? Why is it that python is connected to Mac and all others use c?